be/src/olap/tablet.cpp

```diff
@@ -207,7 +207,7 @@ OLAPStatus Tablet::revise_tablet_meta(const std::vector<RowsetMetaSharedPtr>& ro
 
 OLAPStatus Tablet::add_rowset(RowsetSharedPtr rowset, bool need_persist) {
     DCHECK(rowset != nullptr);
-    WriteLock wrlock(&_meta_lock);
+    WriteLock wrlock(_meta_lock);
```

Comment:
Why do you remove `&` here?

Reply:
Because the mutex and lock of STL are used in this way, I have replaced the mutex defined by doris with the mutex of stl，you can search shared_mutex usage of STL